core/tauri-runtime/src/webview.rs

```diff
@@ -350,6 +350,13 @@ impl WebviewAttributes {
   }
 
   /// Whether page zooming by hotkeys is enabled
+  ///
```

Comment:
We also need to update the documentation in `tooling/api/webview.ts`

Reply:
It only has the zoom API, this shouldn't affect it right?

Reply:
it has `zoomHotkeysEnabled` property when creating new webviews

Reply:
Forgot about that, I'll mirror the docs to their as well